Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts

```diff
@@ -16,17 +16,19 @@ export class AzureAppService {
     private _name: string;
     private _slot: string;
     private _appKind: string;
+    private _isConsumptionApp: boolean;
     public _client: ServiceClient;
     private _appServiceConfigurationDetails: AzureAppServiceConfigurationDetails;
     private _appServicePublishingProfile: any;
     private _appServiceApplicationSetings: AzureAppServiceConfigurationDetails;
 
-    constructor(endpoint: AzureEndpoint, resourceGroup: string, name: string, slot?: string, appKind?: string) {
+    constructor(endpoint: AzureEndpoint, resourceGroup: string, name: string, slot?: string, appKind?: string, isConsumptionApp?: boolean) {
         this._client = new ServiceClient(endpoint.applicationTokenCredentials, endpoint.subscriptionID, 30);
         this._resourceGroup = resourceGroup;
         this._name = name;
         this._slot = (slot && slot.toLowerCase() == constants.productionSlot) ? null : slot;
         this._appKind = appKind;
+        this._isConsumptionApp = isConsumptionApp;
```

Comment:
Make this parameter name more generic like "EnableSyncUpdate" with default value true

Reply:
done..